src/braket/jobs/hybrid_job.py

```diff
@@ -317,7 +320,8 @@ def _log_hyperparameters(entry_point: Callable, args: tuple, kwargs: dict) -> di
             hyperparameters.update(**value)
         else:
             warnings.warn(
-                "Positional only arguments will not be logged to the hyperparameters file."
+                "Positional only arguments will not be logged to the hyperparameters file.",
+                stacklevel=1,
```

Comment:
What does this do?

Reply:
This explicitly sets the stack level so we can be intentional about how much information we provide for debugging. Later on, this will make it easier to alter if more details are needed by customers.

Reply:
OK, but what does 1 mean? Should we be using an enum?